app/services/data/delete-claim-from-external.js

```diff
@@ -7,6 +7,9 @@ module.exports = function (eligibilityId, claimId) {
       return knex('ExtSchema.ClaimDocument').where('ClaimId', claimId).del()
     })
     .then(function () {
+      return knex('ExtSchema.ClaimDocument').where({'ClaimId': null, EligibilityId: eligibilityId}).del()
```

Comment:
replace with a `orWhere` to the existing query to avoid an additional query
http://knexjs.org/#Builder-where